Specs/Number/Swedish/NumberModel.json

```diff
@@ -1376,30 +1358,30 @@
           "value": "0,0476190476190476"
         },
         "Start": 0,
-        "End": 17
+        "End": 16
       }
     ]
   },
   {
-    "Input": "en tjugo femtedel",
+    "Input": "en tjugoförstedels andel",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "en tjugo femtedel",
+        "Text": "en tjugoförstedels",
         "TypeName": "number",
         "Resolution": {
           "subtype": "fraction",
-          "value": "0,04"
+          "value": "0,0476190476190476"
         },
         "Start": 0,
-        "End": 16
+        "End": 17
       }
     ]
   },
   {
     "Input": "tre tjugoförstedelar",
-    "Comment": "Correspons to writing 'Three twentyfirsts'. Compound fraction expressions are currently not handled.",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript, python, java",
+    "Debug": true,
```

Comment:
Please be careful to not leave the "Debug" markers in the specs in the repo.

Reply:
This has been removed